nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/GlassFishMain.java

```diff
@@ -230,25 +239,13 @@ private void doBootCommands(String file) {
             try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
                 
                 System.out.println("Reading in commandments from " + file);
-                String line = reader.readLine();;
+                String line = reader.readLine();
+                CommandRunner cmdRunner = gf.getCommandRunner();
+                
                 while (line != null) {
-                    
-                        String[] commmandParts = cleanCommand(line);
-                        CommandRunner runner = gf.getCommandRunner();
-                        CommandResult result;
-                        if (!(commmandParts[0].isEmpty() || commmandParts[0].equals(" "))){
-                            
-                            if (commmandParts.length == 1){
-                                result = runner.run(commmandParts[0]);
-                            } else {
-                                String[] argv = commmandParts[1].split(" ");
-                                result = runner.run(commmandParts[0], argv);
-                            }
-                            System.out.println(result.getOutput());
-                            if (result.getFailureCause() != null){
-                                throw result.getFailureCause();
-                            }
-                        }
+                    if (!(line.isEmpty() || line.contains("#"))) {
```

Comment:
What about if a line has a command and then a comment afterwards i.e.
set x.y.z=a # this sets a property
The command should run but what comes after the # should be ignored